src/components/error-message/error-message.ts

```diff
@@ -35,12 +36,12 @@ export class MErrorMessage extends ModulVue {
         return window.navigator.userAgent;
     }
 
-    private get dateInfo(): string[] | undefined {
+    private get i18nDate(): string[] | undefined {
         let result: string[] | undefined = undefined;
         if (this.date) {
             result = [this.date.format('YYYY-MM-DD'), this.date.format('HH:mm:ss')];
         }
-        return result;
+        return Vue.prototype.$i18n.translate('m-error-message:date', result, 0, '', true, FormatMode.Default);
     }
```

Comment:
Dans un component on devrait utiliser this.$i18n , pour rester conforme avec le reste et avoir le typing

Reply:
Fait!